packages/priority-queue/README.md

```diff
@@ -0,0 +1,29 @@
+# Priority Queue
+
+This module allows you to run a queue of callback while on the browser's idle time making sure the higher-priority work is performed before.
+
+## Installation
+
+Install the module
+
+```bash
+npm install @wordpress/priority-queue --save
+```
+
+_This package assumes that your code will run in an **ES2015+** environment. If you're using an environment that has limited or no support for ES2015+ such as lower versions of IE then using [core-js](https://github.com/zloirock/core-js) or [@babel/polyfill](https://babeljs.io/docs/en/next/babel-polyfill) will add support for these methods. Learn more about it in [Babel docs](https://babeljs.io/docs/en/next/caveats).
+
+## Usage
+
+```js
+import { createQueue } from '@wordpress/priority-queue';
+
+const queue = createQueue();
+const ctx1 = {};
+const ctx2 = {}; 
+
+queue.add( ctx1, () => console.log( 'This will be printed first' ) );
```

Comment:
Without reading the implementation, I'm not sure what "ctx" abbreviates? An API documentation could help clarify what these arguments are meant to represent.

Reply:
I agree. It feels like something that could be generated from the JS Docs.

Reply:
I tried clarifying with comments.

Reply:
> I agree. It feels like something that could be generated from the JS Docs.

Yes, though this would assume we had JSDocs, which are missing here as well 😄